getting_started.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started
+
+Follow this guide if you want to use Life Monitor to monitor your workflows.
+
+At the moment, the Life Monitor can be used through its [REST API](lm_api_specs)
+or through a command-line client.
+Integration with the [Workflow Hub](https://workflowhub.eu/) is in the works, as
+well as a web interface and a command line client.
+
+To interact with the API, the first thing to do is to first authenticate with the Life
+Monitor.
+
+
+## Create and account and authenticate
+
+Point your browser to the Life Monitor API endpoint <https://api.lifemonitor.eu/>.
```

Comment:
The URL is https://api.dev.lifemonitor.eu/. The screenshot should also change accordingly.

We should also add a note stating that it's a development/testing instance that may disappear (along with any submitted content) at any time.

Reply:
While we currently only have the `dev` instance online, we want to point users to the production instance so I used the appropriate api.lifemonitor.eu URL.  Same goes for the other comments regarding the service address.

Reply:
How about adding something like this to the top of the section then:

"If you just want to play around or get familiar with the API, use the dev instance (replace `api.lifemonitor.eu` with `api.dev.lifemonitor.eu` in the following instructions and examples). Please note that the dev instance is meant for testing / development and it could be wiped out at any time with no warning"

Reply:
done! 2ca5c047469be544ed62b50bf7bc8946738557e9

Reply:
The new text has been added twice. Also, should `:bulb:` and `:warning:` render to something? They are unchanged in https://ilveroluca.github.io/life_monitor/getting_started.

Reply:
yes, you're right.  I enabled the appropriate jekyll plugin and removed the duplicate text.